package.json

```diff
@@ -13,7 +13,7 @@
     "build": "npm run test && npm run clean && npm run lint-js && npm run build-es5 && npm run build-webpack",
     "build-doc": "jsdoc  --package package.json -d doc ./src/**/*.js",
     "build-es5": "babel src --out-dir lib ",
-    "build-webpack": "webpack --progress --colors --display-error-details",
+    "build-webpack": "webpack -p --progress --colors --display-error-details",
     "clean": "rm -rf dist lib doc",
     "dev": "webpack-dev-server --host=0.0.0.0 --inline --config webpack.config.dev.js",
```

Comment:
--linline is now default: https://webpack.js.org/configuration/dev-server/#devserver-inline-cli-only

--host can be specified in the devServer object: https://github.com/cerner/terra-core/blob/master/packages/terra-site/webpack.config.js#L99